sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/ddl.scala

```diff
@@ -43,7 +43,7 @@ import org.apache.spark.sql.types._
 case class CreateTable(
     tableDesc: CatalogTable,
```

Comment:
How can we replace `StringType` without collation in `tableDesc` for v1 command?